nle/nethack/nethack.py

```diff
@@ -192,6 +192,13 @@ def __init__(
         if options is None:
             options = NETHACKOPTIONS
         self.options = list(options) + ["name:" + playername]
+        if next(iter(playername.split("-", 1)[1:]), None) == "@":
```

Comment:
```suggestion
        if playername.endswith("-@"):
```

Reply:
i know the logics not identical, but convince me this wont do

Reply:
I'd like to guard against situations like "Agent-bla-@". I've made that line simpler by doing `if "-" in playername` instead.

Reply:
Now changed to your suggestion of `if playername.split("-", 1)[1:] == ["@"]:`